Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.XamlHost/XamlApplication.cs

```diff
@@ -17,6 +17,12 @@ namespace Microsoft.Toolkit.Win32.UI.XamlHost
     /// </summary>
     internal class XamlApplication : Windows.UI.Xaml.Application, Windows.UI.Xaml.Markup.IXamlMetadataProvider
     {
+        private static readonly List<Type> FilteredTypes = new List<Type>
+        {
+            typeof(XamlApplication),
```

Comment:
It is since https://github.com/Microsoft/WindowsCommunityToolkit/pull/2480/files#diff-0d04f3907fb021b47d22d310da636e99R106 does not make a distinction of if it's an interface and will fail when `Activator.CreateInstance` is called https://github.com/Microsoft/WindowsCommunityToolkit/pull/2480/files#diff-0d04f3907fb021b47d22d310da636e99R108

I only added a comment for that logic to note types of checks that are required (interface, not generic, not abstract, etc.)